ci/ciimage/ubuntu-rolling/install.sh

```diff
@@ -27,6 +27,7 @@ pkgs=(
   bindgen
   itstool
   openjdk-11-jre
+  curl
```

Comment:
Shouldn't that image have wget instead?

Reply:
Indeed, I was using the command from https://rustup.rs/, but changed to wget now, that probably reduce a bit the docker image size.